ckanext/showcase/commands/migrate.py

```diff
@@ -120,3 +131,12 @@ def _find_duplicates(self, lst):
         ['two', 'three']
         '''
         return list(set(x for x in lst if lst.count(x) >= 2))
+
+    def _gen_new_title(self, title):
+        name = munge_title_to_name(title)
+        pkg_obj = model.Session.query(model.Package).filter_by(name=name).first()
+        if pkg_obj:
+            title.replace('duplicate_', '')
+            return 'duplicate_' + title + '_' + str(uuid.uuid4())[:3]
```

Comment:
This isn't unique enough when there are a lot of items to migrate. I did a simple test (5 times) with 400 items:

```python
l = list()
for i in range(0, 400):
   l.append(str(uuid.uuid4())[:3])

len(l) # 400, 400, 400, 400, 400
len(set(l)) # 384, 382, 382, 376, 381
```

Perhaps use the original Related Item id instead of creating another uuid?